sdk/test-utils/perfstress/test/serviceClientGet.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Looks like format step is failing and causing build failures.
Can you run `rushx format` in the `perfstress` folder?